_posts/2021-10-25-waku-v1-vs-waku-v2.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: post
+name:  "Waku v1 vs Waku v2: bandwidth comparison"
```

Comment:
Titlecase: "Bandwidth Comparison" ditto title

Reply:
Done in 21749a4